azure/packages/azure-client/src/test/AzureClient.spec.ts

```diff
@@ -225,7 +204,7 @@ describe("AzureClient", () => {
 
         const { container: containerGet } = await client.getContainer(containerId, schema);
```

Comment:
what events are available on the containerGet object?

Reply:
> what events are available on the containerGet object?

The containerGet is `FluidContainer`, the available events for that are connected/disconnected, disposed, saved and dirty

Reply:
we need to do something about waiting here as i don't trust that the second load of the container will necessarily have all the change of the first available immediately. of the options i think just keep the mapWait is probably the best